run-tests.sh

```diff
@@ -150,5 +148,4 @@ for project in ${PROJECTS[@]}; do
 	rm -rf $BUILD_FOLDER
```

Comment:
This was the source of the problem. The errors.txt file was written in the build folder, and since the build folder was deleted the errors.txt file was also deleted.